webapp/packages/core-connections/src/locales/de.ts

```diff
@@ -62,4 +62,6 @@ export default [
   ['connections_templates_deprecated_message', 'Template connections are deprecated and will be removed in future releases'],
   ['core_connections_connection_driver_not_installed', 'Driver is not installed'],
   ['core_connections_connection_temporary', 'Temporary connection'],
+  ['connections_connection_create_success', 'Connection is created'],
+  ['connections_connection_update_success', 'Connection is updated'],
```

Comment:
please follow naming pattern: `{plugin_name}_...`
in this case is: `core_connections_connection_create_success`